app/controllers/api/v1/rooms_controller.rb

```diff
@@ -58,6 +58,16 @@ def create
         render_json status: :created
       end
 
+      # GET /api/v1/rooms/:friendly_id/recordings.json
+      # Returns: { data: Array[serializable objects] , errors: Array[String] }
+      # Does: gets the recordings that belong to the specific room friendly_id
+      def recordings
+        room_id = Room.find_by(friendly_id: params[:room_friendly_id]).id
+        recordings = Recording.where(room_id:)
```

Comment:
use `find_room` from above -> then you can just do `@room.recordings`

Reply:
The param comes in as ```room_friendly_id```  (rails routes just named it like that automatically) and not ```friendly_id```, that's why I didn't use the ```find_room``` function since it wouldn't work.

I can clean it up to become like this though:
`room = Room.find_by(friendly_id: params[:room_friendly_id]).id`
` recordings = room.recordings`